### PR TITLE
HW 4 done

### PR DESCRIPTION
**Pull request by jigsaff**

### CODE REVIEW

**Comment by igor-budzin** [comment]
I see an error after `npm start` 
image

File: src/components/Dialog/component.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { PortalWithState } from 'react-portal';
+
+export const Dialog = ({ title, children, onClose }) => {
```
Comment (igor-budzin) [inline src/components/Dialog/component.js:4]: You pass `onClose` but not using it anywhere in the component. It should be called on click by close icon

File: src/components/Dialog/component.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import Dialog from './index';
+
```
Comment (igor-budzin) [inline src/components/Dialog/component.test.js:3]: Would be great to add test case for `onClose` callback

**Comment by igor-budzin** [comment]
I have created a [PR](https://github.com/jigsaff/react-gmp/pull/5/files) to demonstrate how to use fous trap and portal together. Feel free to remove this PR after homewok to this task done.

`react-portal` is definately not the best library to manage portals so feel free to use another one or create your own solution in next tasks. But to follow acceptance criteria from learn.portal you need to use it at least in this task

**Comment by jigsaff** [comment]
For some reason this, the simplest solution did not work for me last time.

**Review by igor-budzin — APPROVED** [review]

**Pull request merged**